db/fixtures/questions.yml

```
@@ -66,3 +66,11 @@ question14:
   description: tagが付与された質問2の内容です。
   user: komagata
   practice: practice1
+
+<% 15.upto(50) do |i| %>
+question<%= i %>:
+  title: テストの質問<%= i %>
+  description: ページャー確認用のテスト質問です。
+  user: hajime
+  practice: practice1
+<% end %>
```

```suggestion
<% 16.upto(50) do |i| %>
question<%= i %>:
  title: テストの質問<%= i %>
  description: ページャー確認用のテスト質問です。
  user: hajime
  practice: practice1
  created_at: "2022-01-15"
  published_at: "2022-01-15"
<% end %>
```
私のQ＆AのWIP機能のプルリクを先にマージしていただいたため、コンフリクトが起きてしまいました！
上記の変更でコンフリクトが解消できますので、mainを取り込む前にこちらをご確認いただき、Commit suggestionしてくださいー。

変更点は下記の2点です。
・データ名を16からスタートしていただくこと（私が15を使ってしまったので）、
・今回のwip機能追加でデータには先に`created_at`と`published_at`を設定しておくことが必要になりますので（データの日付はなんでもよい）（この2行がないと質問一覧の表示ができなくなります）、その2行を加えてあります。

私の方がマージが後になると思っていたので、コンフリクトは私が対応するつもりでいました。
お手数をおかけいたしますがよろしくお願いいたします。

---

わざわざコンフリクトの解消コードまで書いていただき、問題なく解消し自分の開発を進めることができました。
大変感謝です、いつもありがとうございます🙏
@Paru871 さんのおかげでコードの提案方法や、Commit suggestionを使えるようになりました！